docs/static_site/Makefile

```diff
@@ -20,7 +20,7 @@ all: html
 html:
 	mkdir -p build
 	cd src && bundle install && JEKYLL_ENV=production bundle exec jekyll build --config _config_prod.yml -d ../build/html && cd ..
-
+	git clone --recursive https://github.com/connorgoggins/mxnet-website-artifacts.git && mv mxnet-website-artifacts/versions/ build/html/ && rm -rf mxnet-website-artifacts
```

Comment:
I suggest to not have artifacts in personal repo. We were planning to keep the artifacts in public readable MXNet assets bucket / similar?

Reply:
I tried storing the artifacts in S3 and pulling them with `wget` in the Makefile, but the Jenkins slave doesn't seem to have the `wget` package installed so I thought just cloning them directly from the repo would be more straightforward. I'll investigate how to get the `wget` package onto the Jenkins slave so we can pull the artifacts from S3 instead.

Reply:
@sandeep-krishnamurthy update: I've incorporated and successfully tested a new round of changes that uses `wget` to pull the artifacts from S3 instead of the personal repo.

Reply:
Thanks.